iothub_client/src/iothubtransport_mqtt_common.c

```diff
@@ -287,12 +284,17 @@ typedef struct MQTT_DEVICE_TWIN_ITEM_TAG
     tickcounter_ms_t msgCreationTime;
     tickcounter_ms_t msgPublishTime;
     size_t retryCount;
-    uint16_t packet_id;
+    uint16_t packet_id; // request_id
```

Comment:
request_id?

Reply:
the packet id correlates to the request id.  It's just not obvious at first glance.